src/box/lua/config/descriptions.lua

```diff
@@ -0,0 +1,2446 @@
+
+    To avoid the split vote repeat, the timeout is randomized on each node
+    during every new election, from 100% to 110% of the original timeout value.
+    For example, if the timeout is 300 ms and there are 3 nodes started the
+    election simultaneously in the same term, they can set their election
+    timeouts to 300, 310, and 320 respectively, or to 305, 302, and 324, and
+    so on. In that way, the votes will never be split because the election on
+    different nodes won't be restarted simultaneously.
+]])
+
+M['replication.failover'] = format_text([[
+    A failover mode used to take over a master role when the current master
+    instance fails. The following modes are available:
+
+    - `off`: Leadership in a replica set is controlled using the
+    `database.mode` option. In this case, you can set the `database.mode` option
+    to rw on all instances in a replica set to make a master-master
+    configuration.
+
+    The default database.mode is determined as follows: `rw` if there is one
+    instance in a replica set; `ro` if there are several instances.
+
+    - `manual`: Leadership in a replica set is controlled using the
+    `<replicaset_name>.leader` option. In this case, a master-master
+    configuration is forbidden.
+
+    In the`manual` mode, the `database.mode` option cannot be set explicitly.
+    The leader is configured in the read-write mode, all the other instances
+    are read-only.
+
+    - `election`: Automated leader election is used to control leadership in a
+    replica set. In the election mode, `database.mode` and
+    `<replicaset_name>.leader` shouldn't be set explicitly.
+    - `supervised`: (Enterprise Edition only) Leadership in a replica set is
+    controlled using an external failover coordinator.
+
+    In the `supervised` mode, `database.mode` and `<replicaset_name>.leader`
+    shouldn't be set explicitly.
+]])
+
+M['replication.peers'] = format_text([[
+    URIs of instances that constitute a replica set. These URIs are used by
+    an instance to connect to another instance as a replica.
+
+    Alternatively, you can use iproto.advertise.peer to specify a URI used to
+    advertise the current instance to other cluster members.
+]])
+
+M['replication.peers.*'] = format_text([[
+    specifies URIs of replica set instances.
+
+    For example: `- replicator:topsecret@127.0.0.1:3301`.
+]])
+
+M['replication.skip_conflict'] = format_text([[
+    By default, if a replica adds a unique key that another replica has added,
+    replication stops with the `ER_TUPLE_FOUND` error. If
+    `replication.skip_conflict` is set to `true`, such errors are ignored.
+]])
+
+M['replication.sync_lag'] = format_text([[
+    The maximum delay (in seconds) between the time when data is written to
+    the master and the time when it is written to a replica. If
+    `replication.sync_lag` is set to `nil` or `365 * 100 * 86400`
+    (`TIMEOUT_INFINITY`), a replica is always considered to be "synced".
+]])
```

Comment:
If `replication.sync_lag` is omitted in the YAML configuration (`nil` is Lua) or set to `null` (`box.NULL` is Lua), it is considered as a default value: 10 seconds.

The `365 * 100 * 86400` value has no special sense. And there is no `TIMEOUT_INFINITY` special value in the YAML configuration.

It seems, it is an incorrect and complicated way to say the following:

> If a replica should remain in the `synched` status disregarding of the network delay, set this option to a large value.

Reply:
Reformatted to 
```lua
I['replication.sync_lag'] = format_text([[
    The maximum delay (in seconds) between the time when data is written to
    the master and the time when it is written to a replica.

    If a replica should remain in the synched status disregarding of the
    network delay, set this option to a large value.
]])
```